client/verta/tests/conftest.py

```diff
@@ -572,8 +577,19 @@ def workspace3(client_sys_admin, created_entities):
 ])
 
 
-def create_workspace(client, created_entities, roles):
-    workspace = client._create_workspace(client._conn._get_organization_id(), generate_default_name(), roles)
+@pytest.fixture
+def workspace4(client_sys_admin, created_entities, namespace):
```

Comment:
I think the other workspace fixtures can get away with numbers because they're kinda generic, but this one's specifically for MNS I think, so highlighting the distinction may be helpful.

```suggestion
def mns_workspace(client_sys_admin, created_entities, namespace):
```